deploy/crds/app.kiegroup.org_kogitodataindices_crd.yaml

```diff
@@ -123,159 +222,99 @@ spec:
                     infrastructure
                   type: boolean
               type: object
-            memoryLimit:
-              description: MemoryLimit is the limit of Memory for the container
-              type: string
-            memoryRequest:
-              description: MemoryRequest is the request of Memory for the container
-              type: string
             replicas:
-              description: Replicas is the number of pod replicas that the Data Index
-                Service will create
+              description: 'Number of replicas that the service will have deployed
+                in the cluster Default value: 1'
               format: int32
-              maximum: 100
+              maximum: 1
```

Comment:
Does it mean that Data index shouldn't scale?

Reply:
@cristianonicolai could you chime in? I don't remember honestly if we will have Jobs Service or Data Index as a single pod in the cluster.

Reply:
job service needs to be single pod for now, scaling needs to be horizontal.
as for data index, it should be fine to scale up.